packages/dd-trace/src/plugins/tracing.js

```diff
@@ -12,6 +12,9 @@ class TracingPlugin extends Plugin {
     this.operation = this.constructor.operation
 
     this.addTraceSub('start', message => {
+      if (message && typeof message === 'object') {
+        message.parentStore = storage.getStore()
```

Comment:
Even if it's an object, we don't want this stored here, unless we know for _sure_ that it's a context object that should die and the end of the span. We also don't want to clobber user objects if we can avoid it.